app/views/layouts/application.html.erb

```diff
@@ -5,7 +5,7 @@
     <%= csrf_meta_tags %>
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
-    <link rel="stylesheet" type="text/css" href="http://fonts.googleapis.com/css?family=Karla">
+    <link rel="stylesheet" type="text/css" href="fonts.googleapis.com/css?family=Karla">
```

Comment:
Should be **//fonts.googleapis.com/css?family=Karla** Just add two slashes. ;) That is because it will automatically detect if we use HTTP or HTTPS. @danTeteruk

Reply:
ok, thanks.

i'll fix it now